projects/lf-documentation/src/app/example-usage-base/example-usage-base.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+import { NavigationExtras, Router } from '@angular/router';
+import { RouterLinks } from '../app-routing.module';
+
+@Component({
+  selector: 'app-example-usage-base',
+  templateUrl: './example-usage-base.component.html',
+  styleUrls: ['./example-usage-base.component.css']
+})
+export class ExampleUsageBaseComponent {
+
+  constructor(public router: Router) { }
+
+  navigateToGettingStarted(id: string) {
```

Comment:
was this created just so you wouldn't have to add this to each class?

could you use a directive instead? or can you not use the router with a directive?